recipes/latte-integrale/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "latte-integrale" %}
+{% set version = "1.7.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/latte-int/latte/releases/download/version_{{ version | replace('.', '_') }}/latte-int-{{ version }}.tar.gz
+  sha256: 733139f206518d0a20f42d1e280428be901f7d29189b2c7845382cbd538953de
+  patches:
+    - gh11.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    # LattE integrale provides a liblatte library which is not very widely used.
+    # Nevertheless it should automatically appear in run when used in host.
+    - {{ pin_subpackage("latte-integrale") }}
+
+requirements:
+  build:
+    - automake
+    - libtool
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - perl
+  host:
+    - ntl
+    - gmp
+    - cddlib
+    # The 4ti2 interface (which only calls the 4ti2 binaries) is only built if
```

Comment:
Actually LattE uses the 4ti2 library, not the binaries, for example in `code/latte/triangulation/RegularTriangulationWith4ti2.cpp`

Reply:
Thanks, fixed in https://github.com/conda-forge/latte-integrale-feedstock/pull/1.